src/components/SearchInput/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
```

Comment:
nit: better to have File name as 
`src/components/SearchInput/SearchInput.tsx`
and have index.ts as: 
` export {SearchInput} from './SearchInput'; `

Reply:
Thanks for your suggestion but I just wanted to keep the same file architecture